packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,17 +1,33 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-shadow: Specifies the shadow of the component.
+ * @prop --calcite-fab-shadow-hover: Specifies the shadow of the component when in hover state.
+ * @prop --calcite-fab-shadow-active: Specifies the shadow of the component when in active state.
+ */
+
 :host {
-  @apply flex bg-transparent;
+  @apply flex;
+  --calcite-fab-shadow: 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08);
+  --calcite-fab-shadow-active: 0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16);
+  --calcite-fab-shadow-hover: var(--calcite-shadow-md);
+  --calcite-fab-offset-shadow: 0 0 #0000;
```

Comment:
☝️ `--calcite-fab-offset-shadow` is missing in the documentation above.

Reply:
I would think this can be an `-internal` prop.

Reply:
Two opposing options, what do you guys think we should go with? Would a user want to be able to change the offset shadow? @macandcheese @eriklharper

Reply:
I'm not sure what this would be used for. Is this instead of a focus override? See this comment about perhaps setting css property overrides on the internal button directly https://github.com/Esri/calcite-design-system/pull/8660#discussion_r1468210782, which I think we should look into.

Reply:
Right now FAB applies its own shadow around the internally rendered Button component. But we didn't have the Button css properties in the past to directly set them on Button. So, I think we can do that now, like what you have above.

But, I don't understand the offset-shadow rule. What is that used for? I'd suggest just getting rid of that if possible. Users should be able to override a focus color to achieve that?

Reply:
@DitwanP @eriklharper offset-shadow should not be a token.